docs/ops/cli.md

```diff
@@ -270,7 +270,9 @@ Action "run" compiles and runs a program.
                                           which to connect. Use this flag to
                                           connect to a different JobManager than
                                           the one specified in the
-                                          configuration.
+                                          configuration. It will be ignored if
```

Comment:
AFAIK this section is generated by the client, and thus the changes must be made to the respective classes. Not sure which though.